config/jobs/kubernetes-sigs/controller-runtime/controller-runtime-presubmits-release-0.16.yaml

```diff
@@ -0,0 +1,34 @@
+presubmits:
```

Comment:
@alvaroaleman @vincepri I don't really mind, but WDTY should we get rid of some of the old pre-submits? We have them for >= release-0.2

Reply:
Yeah we don't need them